packages/verification-vue-component/src/style/components/verification/_file_selection.scss

```diff
@@ -0,0 +1,105 @@
+.verification-dropbox, .verification-file-selector {
+    border-width: $verification-dropbox-border-width;
+    border-style: $verification-dropbox-border-style;
+    border-color: $verification-blue;
+
+    .input-file {
```

Comment:
This is now only in .verification-file-selector

Reply:
right, moved it to .verification-file-selector only